### PR TITLE
Convert SDEC tests to Regression Data

### PR DESCRIPTION
**Pull request by atharva-2001**
### :pencil: Description

**Type:** :vertical_traffic_light: `testing` 
Adds SDEC regression data- https://github.com/tardis-sn/tardis-regression-data/pull/15
Image tests- https://github.com/tardis-sn/tardis-regression-data/pull/17

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: tardis/visualization/tools/tests/test_sdec_plot.py [hunk 1]
```diff
@@ -88,33 +101,50 @@ def sdec_ref_data_path(tardis_ref_path):
 class TestSDECPlotter:
```
Comment (andrewfullard) [inline tardis/visualization/tools/tests/test_sdec_plot.py:101]: Can the `sdec_ref_data_path` function be removed now?

File: tardis/visualization/tools/tests/test_sdec_plot.py [hunk 2]
```diff
@@ -127,6 +157,7 @@ def plotter(self, simulation_simple):
         -------
         tardis.visualization.tools.sdec_plot.SDECPlotter
         """
+        # request.cls.regression_data = RegressionData(request)
```
Comment (andrewfullard) [inline tardis/visualization/tools/tests/test_sdec_plot.py:160]: delete?

File: tardis/visualization/tools/tests/test_sdec_plot.py (hunk 1 above)
Comment (atharva-2001) [reply tardis/visualization/tools/tests/test_sdec_plot.py:80]: Fixed in [`257931c` (#2785)](https://github.com/tardis-sn/tardis/pull/2785/commits/257931c30df41e9a13544c0949e0d686f510a5d3)

File: tardis/visualization/tools/tests/test_sdec_plot.py (hunk 2 above)
Comment (atharva-2001) [reply tardis/visualization/tools/tests/test_sdec_plot.py]: Fixed in [`257931c` (#2785)](https://github.com/tardis-sn/tardis/pull/2785/commits/257931c30df41e9a13544c0949e0d686f510a5d3)

**Review by andrewfullard — DISMISSED** [review]

**Review by jvshields — APPROVED** [review]

**Review by Knights-Templars — APPROVED** [review]

**Pull request merged**